specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/policy.json

```diff
@@ -0,0 +1,1111 @@
+                "schema": {
+                  "$ref": "#/definitions/ErrorResponse"
+                }
+              }
+            }
+          }
+        },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/policysetdefinitions/{policySetDefinitionName}": {
+      "put": {
+        "tags": [
+          "PolicySetDefinitions"
+        ],
+        "operationId": "PolicySetDefinitions_CreateOrUpdate",
+        "description": "Creates or updates a policy set definition.",
+        "x-ms-examples": {
+          "Put a policy set definition": {
+            "$ref": "./examples/createOrUpdatePolicySetDefinition.json"
+          }
+        },
+        "parameters": [
+            {
+              "name": "policySetDefinitionName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the policy set definition to create."
+            },
+            {
+              "name": "parameters",
+              "in": "body",
+              "required": true,
+              "schema": {
+                "$ref": "#/definitions/PolicySetDefinition"
+              },
+              "description": "The policy set definition properties."
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            }
+          ],
+        "responses": {
+            "201": {
+              "description": "Created - Returns information about the policy set definition.",
+              "schema": {
+                "$ref": "#/definitions/PolicySetDefinition"
+              }
+            },
+            "200": {
+              "description": "OK - Returns information about the policy set definition.",
+              "schema": {
+                "$ref": "#/definitions/PolicySetDefinition"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "delete": {
+          "tags": [
+            "PolicySetDefinitions"
+          ],
+          "operationId": "PolicySetDefinitions_Delete",
+          "description": "Deletes a policy set definition.",
+          "parameters": [
+            {
+              "name": "policySetDefinitionName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the policy set definition to delete."
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            }
+          ],
+          "responses": {
+            "204": {
+              "description": "No Content"
```

Comment:
Please provide better descriptions as per recommendation from Gaurav that'd not only help SDK but also better API documentations as well